build/Program.cs

```diff
@@ -30,22 +30,28 @@ public static async Task Main(string[] args)
                 DependsOn(Build),
                 () => RunAsync("dotnet", $"test tests/PseudoLocalizer.Tests/PseudoLocalizer.Tests.csproj -c Release --no-build --verbosity=normal"));
 
-            if (IsTravis() && IsMasterBranch())
-            {
-                Target(
-                    Pack,
-                    DependsOn(Test),
-                    () => RunAsync("dotnet", $"pack src/PseudoLocalizer/PseudoLocalizer.csproj -c Release -o ../../{ArtifactsDir} --no-build"));
+            Target(
+                Pack,
+                DependsOn(Build),
+                () => RunAsync("dotnet", $"pack src/PseudoLocalizer/PseudoLocalizer.csproj -c Release -o ../../{ArtifactsDir} --no-build"));
+
+            Target(
+                Publish,
+                DependsOn(Pack),
+                async () =>
+                {
+                    if (IsTravis() && IsMasterBranch())
```

Comment:
Side note: The third Boolean in the condition in `IsMasterBranch()` seems to be redundant:

```c#
if (!string.IsNullOrWhiteSpace(travisBranch) && travisBranch.Equals("master") && bool.TryParse(travisPr, out bool _))
```

`bool.TryParse(travisPr, out bool _)` is just telling you that there is an env var named `TRAVIS_PULL_REQUEST` and it's value can successfully be parsed to a bool. It does not test whether that value is `true` or `false`. Perhaps you intended to test that the value is `false`? However, I think that would still be redundant, because if the build is for a PR then the branch will not be `master`.

Reply:
I think you're right, but I had some problems detecting the branch and I no longer remember if I used that check for it. I'll try it again to confirm.